lib/rubocop/cop/lint/circular_argument_reference.rb

```diff
@@ -35,6 +35,16 @@ module Lint
       #   def cook(dry_ingredients = self.dry_ingredients)
       #     dry_ingredients.combine
       #   end
+      #
+      #   # bad
+      #   def foo(pie = pie = pie)
+      #     pie.heat_up
+      #   end
+      #
+      #   # bad
+      #   def foo(pie = cake = pie)
+      #     [pie, cake].each(&:heat_up)
+      #   end
```

Comment:
Instead of chaining multiple `bad` examples together, it would be better to contrast `bad` and `good` examples, as in other examples. By doing so, users who encounter a `bad` example will have the opportunity to see the corresponding `good` example that shows how to correct it.